Components/Form.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+function Form() {
+  const [todo, setTodo] = useState("");
+  const [todolist, setTodoList] = useState([]);
+  const [isChecked, setChecked] = useState(false);
+
+  const hanldeChange = (e) => {
+    setTodo(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  const addTodo = () => {
+    //console.log(todo); test
+    if (todo !== "") {
+      const todoInsert = {
+        id: Math.floor(Math.random() * 10000), //Generate random id from 0 to 10000
+        value: todo,
+        isCompleted: false,
+      };
+
+      setTodoList([...todolist, todoInsert]); //... operator "spreads out the own enumerable properties as properties of the element you're creating" stack overflow
+    }
+  };
+
+  const deleteTodo = (e, id) => {
+    e.preventDefault();
+    setTodoList(todolist.filter((t) => t.id != id));
```

Comment:
Always be careful when you use `!=` instead of `!==` since it does not check the type, therefore it can lead to unintended results

Reply:
@BaggioGiacomo please address this one also